src/style.css

```diff
@@ -0,0 +1,93 @@
+html {
+  color: white;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 100px;
+}
+
+.block {
+  display: flex;
+  line-height: 300px;
+  justify-content: center;
+  height: 300px;
+  min-height: 300px;
+  flex-basis: 300px;
+  flex-grow: 1;
+}
+
+.block--1 {
+  background-color: rgba(255,0,0,255);
+}
+
+.block--2 {
+  background-color: rgba(204,0,0,255);
+  order: 1;
+}
+
+.block--3 {
+  background-color: rgba(153,0,0,255);
+  order: 2;
+}
+
+.block--4 {
+  background-color: rgba(102,0,0,255);
+  order: 3;
+}
+
+.block--5 {
+  background-color: rgba(51,0,0,255);
+  order: 4;
+}
+
+.block--6 {
+  background-color: rgba(0,0,0,255);
+  order: 5;
+}
+
+@media (min-width: 600px) {
+  .block--3 {
+    order: 4;
+  }
+
+  .block--4 {
+    order: 3;
+  }
+}
+
+@media (min-width: 900px) {
+  .block {
+    flex-basis: 33%;
+  }
+
+  .block--1 {
+    order: 0;
+  }
+
+  .block--2 {
+    order: 1;
+  }
+
+  .block--3 {
+    order: 2;
+  }
+
+  .block--4 {
+    order: 5;
+  }
+
+  .block--5 {
+    order: 4;
+  }
+
+  .block--6 {
+    order: 3;
```

Comment:
if you look at the result, you will find that the first three elements are located in their places even without the order
write less code